source/Grid/Grid.js

```diff
@@ -389,24 +384,33 @@ export default class Grid extends PureComponent {
    * Useful for animating position changes
    */
   scrollToPosition ({
-    scrollLeft = 0,
-    scrollTop = 0
+    scrollLeft,
+    scrollTop
   } = {}) {
     this._setScrollPosition({ scrollLeft, scrollTop })
   }
 
   /**
-   * Gets a calculated value to be used for `scrollLeft`
+   * Gets offsets for a given cell and alignment
    */
-  getScrollLeft (props = this.props, state = this.state) {
-    return this._getCalculatedScrollLeft(props, state) || 0;
-  }
+  getOffsetForCell ({
+    columnIndex,
+    rowIndex,
+    scrollToAlignment = this.props.scrollToAlignment
+  } = {}) {
+    const scrollToColumn = columnIndex >= 0 ? columnIndex : this.props.scrollToColumn
+    const scrollToRow = rowIndex >= 0 ? rowIndex : this.props.scrollToRow
```

Comment:
Tiny nit: You could set the default `columnIndex` and `rowIndex` values in the same way as you do the `scrollToAlignment` for consistency.